less/button.less

```diff
@@ -8,7 +8,7 @@ span.browserButton {
   cursor: default;
   display: inline-block;
   line-height: 30px;
-  width: 30px;
+  width: 40px;
```

Comment:
Increased to centralize "Cancel", still I think this is not the best solution concerning l10n.